Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/auth/logout/LogoutPage.tsx

```diff
@@ -24,9 +26,11 @@ const LogoutPage = () => {
     useEffect(() => {
         (async () => {
             await logout(null);
+            resetCache();
             await wait(0.7);
             window.location.href = '/';
         })();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
DO you need this?

Reply:
This was something that was present before, but because there was a merge conflict it looks like it's newly added. If removed then eslint requires to add render functions as dependencies when its really not nesessary

Reply:
Let's add them anyway, if they do not introduce unwanted behavior.

Reply:
Adding the `resetCache` function dependency results in multiple logout requests being made

Reply:
![image](https://github.com/SoftUni-Internal/OpenJudgeSystem/assets/19411274/a5e4d73b-8951-439d-9da4-58b7818c2ae1)